big_tests/tests/inbox_SUITE.erl

```diff
@@ -1008,14 +971,14 @@ system_message_is_correctly_avoided(Config) ->
         %% Given a room
         muc_light_helper:given_muc_light_room(Room, Alice, InitOccupants),
         BobRoomJid = <<RoomJid/binary,"/", BobJid/binary>>,
```

Comment:
Aaa, we need a code formatter (I know, it is not related to this PR, but could be fixed in this PR maybe)

```suggestion
        BobRoomJid = <<RoomJid/binary,"/", BobJid/binary>>,
```
```suggestion
        BobRoomJid = <<RoomJid/binary, "/", BobJid/binary>>,
```